ccipcapability/go.mod

```diff
@@ -0,0 +1,229 @@
+module github.com/smartcontractkit/chainlink-ccip/ccipcapability
+
+go 1.24.2
+
+replace github.com/fbsobreira/gotron-sdk => github.com/smartcontractkit/chainlink-tron/relayer/gotron-sdk v0.0.5-0.20250528121202-292529af39df
+
+require (
+	github.com/deckarep/golang-set/v2 v2.8.0
+	github.com/ethereum/go-ethereum v1.15.11
+	github.com/gagliardetto/binary v0.8.0
+	github.com/gagliardetto/solana-go v1.12.0
+	github.com/google/uuid v1.6.0
+	github.com/pkg/errors v0.9.1
+	github.com/smartcontractkit/chain-selectors v1.0.60
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20250616193830-42c18cce582e
+	github.com/smartcontractkit/chainlink-ccip/chains/solana v0.0.0-20250616193830-42c18cce582e
+	github.com/smartcontractkit/chainlink-common v0.7.1-0.20250603190644-0dcaa6d2cc86
+	github.com/smartcontractkit/chainlink-evm v0.0.0-20250617093943-9df97249e67b
+	github.com/smartcontractkit/chainlink-solana v1.1.2-0.20250611193111-99542a0c5d58
+	github.com/smartcontractkit/chainlink/v2 v2.25.0
```

Comment:
We should be able to avoid importing all three of these

Reply:
@jmank88 different teams are implementing CCIP for the different NonEVM chain, so we aren't really involved with how they decide to do it. The CCIP services could be in their own modules though, so there's a path to avoid the dependency in the CCIP protocol layer. They are already using a nice little driver registration service inspired by go's sql package.